server/Html.tsx

```diff
@@ -71,7 +79,11 @@ const Html = (props: Props) => {
 					href="/opensearch.xml"
 				/>
 			</head>
-			<body>
+			<body
+				className={`${props.bodyClassPrefix}-body-wrapper active-target-type-${props
```

Comment:
Prefer the `classNames()` utility to string interpolation for long expressions like this.